### ISSUE TITLE
@-ms-viewport is still an error

### ISSUE THREAD

**Issue by cvrebert** [CSSLint/csslint#443]
Regression of #401? Still seeing it over at Bootstrap:

``` bash
bootstrap (master) $ npm list | grep css
├─┬ grunt-contrib-csslint@0.2.0
│ └─┬ csslint@0.10.0
bootstrap (master) $ grunt csslint
Running "csslint:src" (csslint) task
Linting dist/css/bootstrap.css ...ERROR
[...]
[L7278:C1]
>> Unknown @ rule: @-ms-viewport. This rule looks for recoverable syntax errors. (errors)
[...]
Warning: Task "csslint:src" failed. Use --force to continue.

Aborted due to warnings.
```

**Comment by replete** [comment]
I'm still getting this error in v0.1.5.
```
[12:06:47] [L50:C1] @-o-viewport { Unknown @ rule: @-o-viewport.  (errors)
```
Would be so bad if the inline comment syntax worked better, I'm finding irregular results.

```
/*csslint known-properties:false */
@-ms-viewport {width: device-width;}
@-o-viewport {width: device-width;}
@viewport {width: device-width;}
/*csslint known-properties:true */
```

**Comment by cvrebert** [comment]
@replete Right, it's fixed in `master` but the maintainers haven't made a release version since then; see #514.
